video/out/vo_vaapi_wayland.c

```diff
@@ -30,23 +30,26 @@
 
 #define VA_POOL_NUM_ALLOCATED_INIT 30
 
-struct va_pool_entry {
+struct va_pool_entry
+{
     /* key */
     VASurfaceID surface;
 
     VADRMPRIMESurfaceDescriptor desc;
     struct wl_buffer *buffer;
     struct zwp_linux_buffer_params_v1 *params;
-    uint drm_format;
+    uint32_t drm_format;
 };
-struct va_pool {
+struct va_pool
+{
     struct vo *vo;
     struct va_pool_entry **entries;
-    uint num_entries;
-    uint num_allocated;
+    uint32_t num_entries;
+    uint32_t num_allocated;
```

Comment:
why use fixed width ints here? Nothing depends on their 32-bitness, right? If you do change it back, make sure to also change the iteration variables below

Reply:
This can just be int.